### DISCUSSION TITLE
What should be the default security level for a new account?

### DISCUSSION THREAD

**Discussion by tayler6000** [eigentrustnetwork/ETN#10] [Polls]
See #3 for an explanation of security levels.

**Comment by plexish** [comment]
hm, could you talk me through what I'd see as a user differently between 1 and 2?

**Reply by tayler6000** [reply]
At 2 you would have to enter your password every time you vote for someone.

At 1 you would have to enter your password once per day per computer (and possibly per browser as it's a cookie).

At 0 you need to enter your password to connect the service but that's it.

**Reply by plexish** [reply]
Okay, I vote 0 as that's used in most places around the web, other than securing financial things which we're not.
